.github/workflows/ci.yml

```diff
@@ -237,20 +243,15 @@ jobs:
         with:
           enable-cache: true
 
-      - name: create venv
-        run: uv venv --python 3.13
-
       - name: set up venv
         run: |
+          uv venv
           uv pip install -r pydantic/pyproject.toml --extra timezone
-          uv pip install maturin
-        env:
-          UV_PROJECT_ENVIRONMENT: .venv
+          uv pip install maturin pip
```

Comment:
We shouldn't have to install `pip` here, the make job should use the `uv` environment which has it

Reply:
Here I'm pretty sure we need to install maturin & pip because we manually run it (due to the complexity of the two `uv` environments.)